client/index.html

```diff
@@ -25,7 +25,7 @@
 	<meta name="theme-color" content="#455164">
 
 	</head>
-	<body class="signed-out <%= public ? "public" : "" %>">
+	<body class="signed-out {{ternary public 'public' '' }}">
```

Comment:
OMG Handlebars...
Actually, I found several SO answers that say you can inline `if` like this:

```js
{{ if myBool "True" "False" }}
```

Mind giving it a shot? Equally awkward but we don't have to have a helper just for that then lol...